### PR TITLE
Beta 0.9.0 ---> Staging

### PR DESCRIPTION
**Pull request by oten91**

### CODE REVIEW

**Comment by andrewnguyen22** [comment]
Quick first pass looks good on the Pocket Core side. Going to checkout Tendermint next

File: app/query_test.go [hunk 1]
```diff
@@ -954,3 +954,29 @@ func TestQueryNonExistingAccountBalance(t *testing.T) {
...
+			if tc.upgrades != nil { // NOTE: Use to perform neccesary upgrades for test
+				codec.UpgradeHeight = tc.upgrades.codecUpgrade.height
+				_ = memCodecMod(tc.upgrades.codecUpgrade.upgradeMod)
+			}
+			_, _, cleanup := tc.memoryNodeFn(t, oneAppTwoNodeGenesis())
+			got, err := PCA.QueryAccounts(PCA.LastBlockHeight(), 1, 1)
+			assert.Nil(t, err)
+			assert.NotNil(t, got)
+			assert.NotEqual(t, 1, got.Total)
+
```
Comment (emedrano2112) [inline app/query_test.go:978]: Please remove extra space @oten91

File: app/tx_test.go [hunk 2]
```diff
@@ -915,6 +915,190 @@ func TestChangeParamsSimpleTx(t *testing.T) {
...
+				o, _ := PCA.QueryParam(PCA.LastBlockHeight(), "pocketcore/BlockByteSize")
+				assert.Equal(t, "9000000", o.Value)
+				cleanup()
+				stopCli()
+			}
+		})
+	}
+}
+func TestChangeParamspip22afterActivationHeight(t *testing.T) {
+
```
Comment (emedrano2112) [inline app/tx_test.go:1057]: Please remove extra space

**Review by emedrano2112 — COMMENTED** [review]
Please resolve comments. Everything else LGTM.

**Review by emedrano2112 — COMMENTED** [review]
LGTM

**Pull request merged**